swedish-oidc-profile.md

```diff
@@ -509,24 +504,24 @@ The endpoints described in the Discovery document MUST be secured in accordance
 
 All OpenID Providers are uniquely identified by a URL known as the issuer. This URL serves as the prefix of a service discovery endpoint as specified in \[[OpenID.Discovery](#openid-discovery)\]. 
 
-An OpenID Provider compliant with this profile MUST present a discovery document including the mandatory/required fields specified in section 3 of \[[OpenID.Discovery](#openid-discovery)\] and meet the requirements in the table below: 
+An OpenID Provider compliant with this profile MUST present a discovery document including the required fields specified in section 3 of \[[OpenID.Discovery](#openid-discovery)\] **and** meet the requirements in the table below: 
 
 | Field | Description | Requirement |
```

Comment:
A column called "Requirement" with most values set to "REQUIRED" just feels a bit underspecified. The table in 2.1 states "Support requirement" which feels a bit more clear. Should the column say "Presence" instead?
That is what I suspect is the intended meaning.